### ISSUE TITLE
HDFS supported wheel.

### ISSUE THREAD

**Issue by Franciszzj** [apache/incubator-mxnet#19097]
## Description
A HDFS support wheel. USE_HDFS=1.

**Comment by Xiaoxiong-Liu** [comment]
Has this feature been implemented now?

**Comment by leezu** [comment]
No. But contributions are welcome. You can add code at https://github.com/apache/incubator-mxnet/tree/master/tools/dependencies to create static build of the hdfs library and modify the config files in https://github.com/apache/incubator-mxnet/tree/master/config/distribution to enable hdfs for the wheel build.

**Comment by szha** [comment]
@leezu I don't see USE_HDFS flag in the CMakeLists.txt. Where will such flag be passed in cmake?

**Comment by leezu** [comment]
https://github.com/dmlc/dmlc-core/blob/4cabfeafb4f8f8e574de89cea0780c993c20c2d8/CMakeLists.txt#L16
